modules/violations-view-deployment-tab.adoc

```diff
@@ -29,17 +30,34 @@ The overview section lists the following information:
 
 The container configuration section lists the following information:
 
-* *Image Name*: The name of the image for the selected deployment.
+* *Image Name*: The name of the image for the selected deployment. Click the name to view more information about the image.
 * *Resources*:
-** *CPU Request (cores)*: The number of cores requested by the container.
-** *Memory Request (MB)*: The memory size requested by the container.
+** *CPU request (cores)*: The number of cores requested by the container.
+** *CPU limit (cores)*: The maximum number of cores that can be requested by the container.
+** *Memory request (MB)*: The memory size requested by the container.
+** *Memory limit (MB)*: The maximum memory that can be requested by the container.
 * *Volumes*:
 ** *Name*: The name of the location where the service will be mounted.
 ** *Source*: The data source path.
 ** *Destination*: The path where the data is stored.
 ** *Type*: The type of the volume.
 * *Secrets*: Secrets associated with the selected deployment.
 
+[discrete]
+== Port configuration section
+
+* *ports*:
+* *containerPort*:
+* *protocol*:
+* *exposure*:
+* *exposureInfo*:
+** *level*:
+** *serviceName*:
+** *serviceID*:
+** *serviceClusterIp*:
```

Comment:
The IP address another deployment or service _within the cluster_ can use to reach the service. This is not the external IP